client/src/Views/Home/Steps/AddStep.js

```diff
@@ -0,0 +1,159 @@
+import React, { useState } from "react";
+import {
+  Grid,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+  makeStyles
+} from "@material-ui/core";
+import { EditorState, convertToRaw, Modifier } from "draft-js";
+
+import { addStepToCampaign } from "Utils/api";
+import TextEditor from "Components/TextEditor/TextEditor";
+import StepHeader from "Components/TextEditor/StepHeader";
+import StepFooter from "Components/TextEditor/StepFooter";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    fontFamily: "'Open Sans', sans-serif",
+    justifyContent: "center"
+  },
+  // remove scrollbars
+  dialogContent: {
+    maxWidth: "900px",
+    overflow: "hidden",
+    height: "100%",
+    padding: "1rem"
+  },
+  emailContainer: {
+    margin: "1rem 2rem"
+  }
+}));
+
+const Step = props => {
```

Comment:
This component currently shouldn't exist. This looks like @umbauk 's previous implementation of the text editor, which I renamed to `StepEditor` and moved to `/Views/Home/Campaigns/Campaign`

Reply:
Hmmm, maybe I didn't merge dev before I pushed. I'll check it out